python/ray/dag/tests/experimental/test_torch_tensor_dag.py

```diff
@@ -100,6 +104,72 @@ def forward(self, inp):
         return torch.randn(10, 10)
 
 
+class TestNcclGroup(GPUCommunicator):
```

Comment:
We can move this back to each test for now, because I can't pass tests when it's outside. I suspect there might be some environment problems.

Reply:
ok